src/Comments/CommentList/index.tsx

```diff
@@ -15,3 +20,82 @@ export function CommentList({ comments }: CommentListProps) {
         </div>
     );
 }
+
+export function CommentListWithLinks({ comments }: CommentListProps) {
+    return (
+        <div className={s.container}>
+            {comments.map(comment => {
+                return <CommentWithLink key={comment.id} comment={comment} />;
+            })}
+        </div>
+    );
+}
+
+function CommentWithLink({ comment }: { comment: Comment }) {
+    const getDocument = useDocument(comment.ref);
+    const linkedDocument = getDocument();
```

Comment:
This could be deleted. In that case the whole page would turn into a not found – so better use `getDocument(true)`.

Reply:
just happened to me locally, fixed this way https://github.com/macht-sprache/macht-sprache-app/commit/35df8d12567efddfa7ac0e8618a90ceba4683f52